cpp/include/cudf/io_types.h

```diff
@@ -43,4 +43,6 @@ typedef struct
     const char* na_rep;           // string to use for null entries
     bool include_header;          // Indicates whether to write headers to csv
 
+    int chunk_rows;               // maximum number of rows to process for each file write
```

Comment:
The naming of this variable doesn't match the description. `chunk_rows` sounds like a true or false option parameter, not a "maximum". Perhaps `rows_per_chunk` would be clearer?

Reply:
Yes, I like `rows_per_chunk` better.